datacapture/src/test/java/com/google/android/fhir/datacapture/validation/DecimalTypeMaxDecimalValidatorTest.kt

```diff
@@ -34,16 +34,15 @@ import org.robolectric.annotation.Config
 @RunWith(RobolectricTestRunner::class)
 @Config(sdk = [Build.VERSION_CODES.P])
 class DecimalTypeMaxDecimalValidatorTest {
-
   lateinit var context: Context
 
   @Before
-  fun initContext() {
+  fun setupTest() {
```

Comment:
conventionally we call this `setUp`

```suggestion
  fun setUp() {
```

Reply:
hm i realised this is the case for the other validator tests.... if you don't mind renaming all of them to `setUp` it'd be good. thanks!